.github/workflows/deploy.yml

```diff
@@ -47,34 +55,47 @@ jobs:
     runs-on: 'ubuntu-latest'
     steps:
       - name: Check out repository
+        if: inputs.deploy_type == 'stable'
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # https://github.com/actions/checkout/releases/tag/v3.5.3
-      - name: Set preview in package.json
+
+      - name: Check out prerelease
+        if: inputs.deploy_type == 'prerelease'
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # https://github.com/actions/checkout/releases/tag/v3.5.3
+        with:
+          ref: 'f-tfc'
```

Comment:
```suggestion
```

The `workflow_dispatch` type of event trigger actually implies the select box for branches and the `checkout` action already seems to reflect that, so we shouldn't need to do anything differently here.

![Screenshot 2023-06-23 at 14 41 37](https://github.com/hashicorp/vscode-terraform/assets/287584/b02c0fa8-94de-4d1e-97bb-df72e5628303)

We just need to make sure we read the chosen branch name where appropriate, e.g. from `$GITHUB_REF_NAME` variable.